src/store/reducers/executeQuery.js

```diff
@@ -39,7 +39,7 @@ const initialState = {
                 ? MAXIMUM_QUERIES_IN_HISTORY - 1
                 : queriesHistoryInitial.length - 1,
     },
-    runAction: RUN_ACTIONS_VALUES.script,
+    runAction: undefined,
```

Comment:
I seems that `runAction` should be taken from userSettings on application initialisation.